website/docs/cli/commands/vendor/vendor-pull.mdx

```diff
@@ -108,6 +108,112 @@ The following config can be used to download the `vpc` component from an AWS pub
       version: "latest"
   ```
 
+## Vendoring from SSH
+
+This is the most common way to access non-public Git repositories from automated systems because it allows access to private repositories without interactive prompts.
+Atmos will use any installed SSH keys automatically. There are two primary ways to specify an SSH source:
+
+### Implicit SSH
+
+When no explicit scheme is provided, for example:
+
+```shell
+git::git@github.com:cloudposse/terraform-null-label.git?ref={{.Version}}
+```
+
+Atmos’s rewrites this URL to:
+
+```shell
+git::ssh://git@github.com/cloudposse/terraform-null-label.git?depth=1&ref={{.Version}}
+```
+
+In this case, Atmos injects a default username ("git").
+
+### Explicit SSH
+
+When the SSH scheme is specified explicitly, for example:
+
+```shell
+git::ssh://git@github.com/cloudposse/terraform-null-label.git?ref={{ .Version }}
+```
+
+the URL is used as provided. In this scenario, no rewriting occurs because the URL already includes an explicit `ssh://` schema.
+
+### Important Notes
+
+- A URL like:
+  ```shell
+  github.com:cloudposse/terraform-null-label.git?ref={{ .Version }}
+  ```
+  is **not valid** because go-getter misinterprets **github.com:** as a schema and will error out.
```

Comment:
```suggestion
  is **not valid** because go-getter misinterprets **github.com:** as a schema and will error out. To fix this, add the `git::` prefix to the source.
```